internal/appsec/listener/grpcsec/grpc.go

```diff
@@ -117,6 +118,10 @@ func (l *wafEventListener) onEvent(op *types.HandlerOperation, handlerArgs types
 		httpsec.RegisterRoundTripperListener(op, &op.SecurityEventsHolder, wafCtx, l.limiter)
 	}
 
+	if l.isSecAddressListened(ossec.ServerIOFSFileAddr) {
+		ossec.RegisterOpenListener(op, &op.SecurityEventsHolder, wafCtx, l.limiter)
+	}
```

Comment:
Would be best to match the other listeners pattern, and have an `ossec.OSAddressesPresent` function, so the listing of supported addresses is centralized.